src/sparseml/pytorch/image_classification/utils/helpers.py

```diff
@@ -344,17 +345,24 @@ def get_dataset_and_dataloader(
 # Model creation Helpers
 def create_model(
```

Comment:
I am folding alternative pathway for inferencing `num_classes` if the user does not wish to provide it themselves. `num_classes` needs to be specified to create a model.

Reply:
(see: `src/sparseml/pytorch/image_classification/export.py`)